<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_start">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt Medium</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#button_start{
background-color: rgb(240,240,240);
border-radius: 70px;
border: 2px solid #00ccff;
color: #00ccff;
}

#button_start:hover{
 border: 2px solid rgb(180,180,180);
color: rgb(180,180,180);
}</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>91</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="button_time_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#button_time_2{border-image: url(icon/icon_blue-03.png);
}


#button_time_2:hover{
background-color:rgba(250, 250, 250,150);

}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_blink_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(icon/icon_blue-04.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_bright_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(icon/icon_blue-09.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_age_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(icon/icon_blue-08.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_sex_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(icon/icon_blue-07.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_distance_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(icon/icon_blue-06.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_angle_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(icon/icon_blue-05.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>371</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);
</string>
    </property>
    <widget class="QLabel" name="label_time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>29</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00ccff;
border-bottom: 1px solid #00ccff;
font: 13px arial, sans-serif;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_blink">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00ccff;
border-bottom: 1px solid #00ccff;
font: 13px arial, sans-serif;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_angle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00ccff;
border-bottom: 1px solid #00ccff;
font: 13px arial, sans-serif;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_distance">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00ccff;
border-bottom: 1px solid #00ccff;
font: 13px arial, sans-serif;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_sex">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00ccff;
border-bottom: 1px solid #00ccff;
font: 13px arial, sans-serif;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_age">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00ccff;
border-bottom: 1px solid #00ccff;
font: 13px arial, sans-serif;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_brightness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00ccff;
border-bottom: 1px solid #00ccff;
font: 13px arial, sans-serif;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>button_start</zorder>
   <zorder>widget</zorder>
  </widget>
 </widget>
 <resources>
  <include location="iconnew.qrc"/>
 </resources>
 <connections/>
</ui>
